File: java/Table.java
// kelas tabel, untuk membuat dan menampilkan tabel
class Table {
    // method border untuk menampilkan border dengan simbol yang diinginkan
    public static void border(int length, char symbol) {
        // tentukan panjang border dan simbol yang diinginkan
        for(int i=0; i<length; i++){
            // looping simbol hingga membentuk baris
            System.out.print(symbol);
        }
        System.out.println();
    }
    
    // method border2 untuk menampilkan border yang sudah ditentukan
    public static void border2(int length[], char symbol) {
        // parameter dengan panjang dan simbol yang diinginkan
        for(int j=0; j<4; j++){
            // loop sebanyak kolom tabel
            System.out.print("+");
            for(int i=0; i<length[j]-1; i++){
                // loop simbolnya sebanyak panjang length
                System.out.print(symbol);
            }
        }
        System.out.println("+");
    }
}

File: py/Table.py
# inport liblary
import sys

# Kelas Tabel untuk membuat dan menampilkan tabel
class Table:
    # Metode untuk menampilkan border dengan simbol tertentu
    def border(self, panjang, simbol):
        # Menentukan panjang border dan simbol yang akan digunakan
        for i in range(panjang):
            # Melakukan looping pada simbol untuk membentuk baris
            sys.stdout.write(simbol)
        sys.stdout.write("\n")

    # Metode untuk menampilkan border dengan panjang dan simbol tertentu
    def border2(self, panjang, simbol):
        # Parameter dengan panjang dan simbol yang akan digunakan
        for j in range(4):
            # Melakukan looping sebanyak jumlah kolom dalam tabel
            sys.stdout.write("+")
            for i in range(panjang[j]-1):
                # Melakukan looping pada simbol sesuai dengan panjang yang diberikan
                sys.stdout.write(simbol)
        sys.stdout.write("+\n")